### ISSUE TITLE
Feature Request: Read Only Mode

### ISSUE THREAD

**Issue by walles** [atlassian/github-for-jira#843]
Greetings, Atlassian!

When I tried to activate this plugin it asked for write permissions to my github repos.

I want to give it read permissions only.

How can I run this plugin in read-only mode?

**Comment by mboudreau** [comment]
Hi!

We have a ticket in our backlog to investigate this further.  We had tried to use read-only mode, but we did get some issues in some places where it didn't work.  We need to look further into why that is with Github.

**Comment by rachellerathbone** [comment]
Hey @walles. We don't have a read-only mode but I thought I'd let you know that we've changed our permissions so we now no longer require any access to actions and only read permissions for deployments. I've also updated the permissions section of our [README](https://github.com/atlassian/github-for-jira/blob/main/README.md) and added an [FAQs](https://github.com/atlassian/github-for-jira/blob/main/docs/FAQs.md) doc that the team plans to add to when we see questions being commonly asked by our users.
